### DISCUSSION TITLE
Processing the users data into metrics

### DISCUSSION THREAD

**Discussion by diegoq48** [uprm-inso4101-2023-2024-s2/semester-project-personalfinancemanager#8] [Ideas]
How much data do we need in order to generate insightful metrics into the users finance what metrics would we be using and how would they be displayed?

**Comment by AngelCIICMorales** [comment]
We need to know how much money the user has, how much they are spending, and how they want to split their spending. We can use pie charts, bars, or a graph showing the money earned vs money spent. Perhaps the app can have different settings depending on how the user wants the data to be visualized.

**Reply by diegoq48** [reply]
Remember that the user will be inputing the data individually as in for every transaction so you'll have to account for the fact that ever transaction will have a entry in the db other than that it lgtm lmk what do you need in terms of time and manpower

**Reply by AngelCIICMorales** [reply]
Maybe around two weeks to get make something worthwhile with a team of no less than 5 people.

**Reply by diegoq48** [reply]
Remeber that for now we're just fighting for the deadline so we dont need something super polished rather we just need something functional to show off so worry more about having it just work rather than it being polished open the issue and ill start assigning
